_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+| Category      | Evaluation  | Weight  |
+|---|---|---|
+|Price          | 1 to 5 | 50%  |
+|Delivery Time  | 1 to 5 | 20%  |
+|Warranty Period| 1 to 5 | 30%  |
+
+The selected vendor will therefore be the one with the closest score to 100.
+
+Establishing the evaluation factors up-front is also a technique to structure your requests and communicate to vendors how their answers should be formatted. The more technical the request is (RFQ in particular), the more precise the evaluation factors are going to be, and the last thing you want when comparing proposals is to spend hours dissecting a proposal to identify factors impacting the scoring.
+
+Less technical requests (RFIs for example), could be evaluated on additional criteria such as:
+
+ - Quality of the technical response
+ - Technical expertise of the vendor's personnel (based on their bios, certifications ...)
+ - Experience of the vendor in delivering similar projects
+
+Keep in mind, while preparing the request and evaluating responses that the less technical the request and evaluation criteria is, the harder it becomes to evaluate and justify such an evaluation. So chose your evaluation criteria wisely to serve your project's purpose and facilitate the procurement process, in particular if a vendor decides to challenge the award decision.
+
+# Our strategy
+
+As mentioned above, the RFSQ appeared to be the ideal method for Collaboratory procurement, and we used the initial purchase as the baseline to build the RFSQ and to pre-select vendors.
+
+Having an RFSQ signed with vendors allows us to focus our subsequent requests on the technical specifications, maintaining the common contractual framework (support, RMA process) negotiated initially. Each purchase as part of the RFSQ is then a simple RFQ containing detailed specifications of products to be purchased, with the biggest factor impacting the score being pricing.
+
+It is worth mentioning that, at any point in time, we could make purchases outside of the RFSQ framework, as long as we do so following [procurement policies](http://www.procurement.utoronto.ca/programs-and-services/purchasing-goods-services).
+
+# ... Four purchases later
+
+Until now, we've been running 4 RFQ under this RFSQ framework, and are likely going to run a last one by the end of the project's initial grant period. Proceeding with an RFSQ provided us with a relative quick turnover and simplified quotation process while following UofT procurement policies.
+
+<center>
+  <figure style="width: 70%;">
+      <img src="{{site.urlimg}}francois_gerthoffert/collab-procurement/collaboratory-rack-second-first-purchase.png"/>
+      <figcaption>Collaboratory infrastructure after our first purchase</figcaption>
+  </figure>
+</center>
+
+
+But did our initial strategy worked out, and effectively provided us with more resources per dollar spent ?
```

Comment:
But did our initial strategy work out, and effectively provide us with more resources per dollar spent?